ArduCopter/version.cpp

```diff
@@ -0,0 +1,16 @@
+#include "version.h"
+
+#include "GCS_MAVLink/GCS.h" // for FIRMWARE_VERSION_TYPE_DEV and friends
+
+const AP_FWVersion fwver
```

Comment:
it seems to be to allow taking a reference in generic code by using the extern declaration.